compiler-rs/compiler-wasm-lib/src/lib.rs

```diff
@@ -16,12 +16,11 @@
 // under the License.
 
 use anyhow::bail;
-use wasm_bindgen::prelude::*;
 use clients_schema::{Availabilities, Visibility};
+use wasm_bindgen::prelude::*;
 
-
-#[cfg(not(target_arch="wasm32"))]
-compile_error!("To build this crate use `make compiler-wasm-lib`");
+// #[cfg(not(target_arch = "wasm32"))]
+// compile_error!("To build this crate use `make compiler-wasm-lib`");
```

Comment:
```suggestion
#[cfg(all(not(feature = "cargo-clippy"), not(target_arch = "wasm32")))]
compile_error!("To build this crate use `make compiler-wasm-lib`");
```

This should make that compatible with clippy while retaining the warning against direct compilation.